lib/src/main/kotlin/com/github/bratek20/hla/generation/impl/core/api/ComplexStructureField.kt

```diff
@@ -35,7 +35,13 @@ open class ComplexStructureField(
     }
 
     fun exampleValue(): String? {
-        return def.getAttributes().firstOrNull { it.getName() == "example" }?.getValue()
+        val typeName = def.getType().getName()
+        val intBaseType = BaseType.INT.name.lowercase()
+        val longBaseType = BaseType.LONG.name.lowercase()
+        if(typeName ==  intBaseType || typeName == longBaseType) {
```

Comment:
the better check would be done if you would check if type is BaseApiType and then extract BaseType from BaseApiType and then you could compare BaseType enums instead of strings

Reply:
changed as requested, thx for showing it